drizzlepac/devutils/disp_sources_by_flag.py

```diff
@@ -46,14 +47,15 @@ def display_regions(imgname,reg_dict,flag_counts,n_sources):
 
     for ctr in range(0, len(bit_list)):
         bit_val = bit_list[ctr]
-        padding0 = 2 - len(str(ctr+1))
+        # padding0 = 2 - len(str(ctr + 1))
         padding1 = 6 - len(str(bit_val))
         padding2 = 27 - len(flag_meanings[ctr])
-        print("Frame {}: Bit value {}{}{}{}{} sources flagged ({}% of all sources)".format(ctr+1,bit_val,"."*padding1,
-                                                                                           flag_meanings[ctr],
-                                                                                           padding2*".",
-                                                                                           flag_counts[ctr],
-                                                                                           100.0*(float(flag_counts[ctr])/float(n_sources))))
+        print("Frame {}: Bit value {}{}{}{}{} sources flagged ({}% of all sources)".format(ctr + 1,
+                            bit_val, "." * padding1,
+                            flag_meanings[ctr],
+                            padding2 * ".",
+                            flag_counts[ctr],
+                            100.0 * (float(flag_counts[ctr]) / float(n_sources))))
         if ctr != 0:
```

Comment:
Is it possible for n_sources to be zero?

Reply:
This routine gets called directly (mostly interactively) by the user when they have sources they want to display, and 'n_sources' is one of the parameters provided by the user.  So, I would say it is not likely, simply due to the fact that if 'n_sources' was 0, there would be no need to call this code in the first place.